src/app/pages/request-detail/request-detail.component.ts

```diff
@@ -84,7 +84,15 @@ export class RequestDetailComponent implements OnInit {
   }
 
   confirmRemove() {
-    this.removal_content = "Removal triggered."
+    this.removal_content = "Removal triggered.";
+
+    this.requestService.removeRequest(this.request_id).subscribe(response => {
+      if(response.success) {
+
```

Comment:
what is this? why do we have it as if and else wthout doing anything?

Reply:
this.requestService.removeRequest(this.request_id).subscribe(response => {
      if(response.success) {
        this.removal_content = "Request detail successfully deleted.";
      } else {
        this.removal_content = "Failed while deteting request detail.";

Reply:
this hat i'm trying to do